app/views/content/life-as-a-teacher/pay-and-benefits/bens-favourite-things-about-teaching/_article.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="row inset">
+  <section class="col col-720 col-space-l-top">
+      <p>What I particularly love about my school is the excitement and energy I can feel in the corridors. My students are enthusiastic and they push me to understand more about physics and maths too. It also helps that my colleagues are really friendly and supportive. I know that everyone is pulling in the same direction, and we have a good joke in the office.</p>
```

Comment:
Same indenting here as the other html files. `<p>` should be directly under `<section>`